src/util.jl

```diff
@@ -172,7 +173,16 @@ rms(s::AbstractArray{T}) where {T<:Number} = sqrt(sum(abs2, s)/length(s))
 # root mean square of fft of signal
 rmsfft(f::AbstractArray{T}) where {T<:Complex} = sqrt(sum(abs2, f))/length(f)
 
+function meanfreq{T}(f::AbstractVector{T},ΔT)
```

Comment:
Should use `where T` syntax (or not introduce `T` a all). Also probably `where T<:Real` would be in order, no?
Also, if we want to match Matlab, the second argument should be the sampling rate rather than the sampling interval. And a default of 2π would be nice to get normalized frequency by default.